cli/integrationtest/migrations_test.go

```diff
@@ -139,53 +139,55 @@ func getTempDockerInstallationPath(t *testing.T) string {
 	return installationPath
 }
 
-func TestDockerOnlineMigrations(t *testing.T) {
-	t.Parallel()
-	skipUnlessUsingUnixSocket(t)
-	skipIfNotUsingUnixSocketDirectly(t)
-
-	installationPath := getTempDockerInstallationPath(t)
-	defer os.RemoveAll(installationPath)
-
-	environmentConfig := runCommandSucceeds(t, "../../scripts/get-config.sh", "--docker")
-
-	lowercaseTestName := strings.ToLower(t.Name())
-	if len(lowercaseTestName) > 23 {
-		lowercaseTestName = lowercaseTestName[:23]
-	}
-	configMap := make(map[string]any)
-	require.NoError(t, yaml.Unmarshal([]byte(environmentConfig), &configMap))
-	configMap["environmentName"] = lowercaseTestName
-	configMap["installationPath"] = installationPath
-	configMap["initialDatabaseVersion"] = 3
-	p, err := dataplane.GetFreePort()
-	require.NoError(t, err)
-	configMap["dataPlanePort"] = p
-	configMap["network"] = map[string]any{"subnet": "172.255.0.0/24"}
-
-	updatedEnvironmentConfigBytes, err := yaml.Marshal(configMap)
-	require.NoError(t, err)
-
-	tempDir := t.TempDir()
-	configPath := fmt.Sprintf("%s/environment-config.yaml", tempDir)
-	require.NoError(t, os.WriteFile(configPath, updatedEnvironmentConfigBytes, 0644))
-
-	tygerPath, err := exec.LookPath("tyger")
-	require.NoError(t, err)
-	tygerPath, err = filepath.Abs(tygerPath)
-	require.NoError(t, err)
-
-	defer func() {
-		runCommandSucceeds(t, "sudo", tygerPath, "api", "uninstall", "-f", configPath, "--delete-data", "--preserve-run-containers")
-	}()
-
-	runCommandSucceeds(t, "sudo", tygerPath, "api", "install", "-f", configPath)
-
-	runTygerSucceeds(t, "api", "migrations", "apply", "--latest", "--offline", "--wait", "-f", configPath)
-
-	logs := runTygerSucceeds(t, "api", "migrations", "logs", "4", "-f", configPath)
-	assert.Contains(t, logs, "Migration 4 complete")
-}
+// TODO: Re-enable this test when we have a current database version that supports online migrations
```

Comment:
This disabling tests for a given version could be "formalized" with some kind of flag that skips tests when current version doesn't support online migration, but probably not really worth it.